service/messageHandler.go

```diff
@@ -5,21 +5,24 @@ import (
 	"regexp"
 	"time"
 
+	log "github.com/Financial-Times/go-logger"
 	"github.com/Financial-Times/kafka-client-go/kafka"
 	"github.com/satori/go.uuid"
-	log "github.com/sirupsen/logrus"
 )
 
 const messageTimestampDateFormat = "2006-01-02T15:04:05.000Z"
 
+const mapperEvent = "Map"
+const annotationsContentType = "annotations"
+
```

Comment:
can you rename this annotationType or something like that  it's not a content type.

Reply:
Fixed!